.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: node_js
```

Comment:
check how to differ the publish between dev and master. dev should add "-beta" after the version